src/main/java/com/nhnacademy/hello/controller/order/OrderController.java

```diff
@@ -67,6 +73,44 @@ public ResponseEntity<?> returnOrder(
             @RequestBody ReturnsRequest returnsRequest
             ){
 
+        // 반품 사유 조회
+        ReturnsReasonDTO returnsReasonDTO = returnsReasonAdapter.getReturnsReasonById(returnsRequest.reasonId());
+
+
+        // 결제 정보 조회
+        TossPaymentDto payment = tossPaymentAdapter.getPayment(orderId).getBody();
+
+        // 토스 주문 취소 처리
+        ResponseEntity<?> response = tossService.cancelPayment(
+                payment.paymentKey(),
+                "구매자 반품 처리: " + returnsReasonDTO.returnsReason(),
+                payment.amount() - 3000 // 배송비 제외한 금액 환불
```

Comment:
만약 payment.amount()가 3000 이하인 경우 어떻게 되나요?
```suggestion
                payment.amount() - 3000 // 배송비 제외한 금액 환불
```

Reply:
3000 이상인 경우 과정에서 환불은 하지 않도록 수정했습니다